src/lib/transition_frontier_controller/transition_frontier_controller.ml

```diff
@@ -40,17 +48,26 @@ module Make (Inputs : Inputs_intf) :
    and type syncable_ledger_query := Inputs.Syncable_ledger.query
    and type syncable_ledger_answer := Inputs.Syncable_ledger.answer = struct
   open Inputs
+  open Consensus_mechanism
 
-  let run ~transition_reader ~sync_query_reader ~sync_answer_writer =
+  let run ~genesis_transition ~transition_reader ~sync_query_reader ~sync_answer_writer =
     let valid_transition_reader, valid_transition_writer =
       Linear_pipe.create ()
     in
     let catchup_job_reader, catchup_job_writer = Linear_pipe.create () in
-    let frontier = Transition_frontier.create () in
-    Transition_handler.Validator.run frontier ~transition_reader
-      ~valid_transition_writer ;
-    Transition_handler.Processor.run frontier ~valid_transition_reader
-      ~catchup_job_writer ;
-    Catchup.run frontier ~catchup_job_reader ;
-    Sync_handler.run frontier ~sync_query_reader ~sync_answer_writer
+    (* TODO: initialize transition frontier from disk *)
```

Comment:
For now we can just initialize it with the single best as the root like we did before. This will get us to integration test faster